monai/apps/pathology/transforms/post/array.py

```diff
@@ -458,6 +458,8 @@ def __call__(self, contours: List[np.ndarray]) -> np.ndarray:
                     elif coord[1] == self.width - 1:
                         corner = 2
                         pixel = (int(coord[0] - 0.5), int(coord[1]))
+                    else:
+                        raise ValueError("Invalid coutour is generated, may contain boundaries or small holes")
```

Comment:
Could print more details such as coord in the error msg

Reply:
Thanks! Have added!